python/vyos/accel_ppp_util.py

```diff
@@ -144,6 +144,10 @@ def verify_accel_ppp_base_service(config, local_users=True):
             if "key" not in radius_config:
                 raise ConfigError(f'Missing RADIUS secret key for server "{server}"')
 
+    if dict_search('authentication.radius.dynamic_author.server', config):
+        if not dict_search('authentication.radius.dynamic_author.key', config):
+            raise ConfigError('DA/CoE server key required!')
```

Comment:
Typo, I think there should be `DAE/CoA` (Dynamic Authorization Extension/Change of Authorization server)

Reply:
done